src/applications/survivors-benefits/config/form.js

```diff
@@ -99,6 +105,73 @@ const formConfig = {
         },
       },
     },
+    // Chapter 7
+    additionalInformation: {
+      title: 'Additional information',
+      pages: {
+        directDeposit: {
+          title: 'Direct deposit for survivor benefits',
+          path: 'additional-information/direct-deposit',
+          uiSchema: directDeposit.uiSchema,
+          schema: directDeposit.schema,
+        },
+        directDepositAccount: {
+          title:
+            directDepositAccount.uiSchema['ui:title'] ||
+            'Account information for direct deposit',
+          path: 'additional-information/direct-deposit/account',
+          depends: directDepositAccount.depends,
+          uiSchema: directDepositAccount.uiSchema,
+          schema: directDepositAccount.schema,
+        },
+        otherPaymentOptions: {
+          title:
+            otherPaymentOptions.title ||
```

Comment:
Can you explain how this or the path could change? It might be explained later in tests, but I am not sure what this is doing.